plugins/org.jkiss.dbeaver.ext.postgresql.ui/OSGI-INF/l10n/bundle_pt_BR.properties

```diff
@@ -1,33 +1,136 @@
+### O intuito deste explicativo de é manter consistência na tradução do DBeaver para o português brasileiro.
+### Nada aqui é imposição, são apenas sugestões. Se tiver ideias melhores de como expressar uma tradução, sinta-se
+### à vontade para alterar o que for necessário. Apenas atualize as definições abaixo conforme sua nova tradução.
+
+### Em várias ocasiões a tradução de uma frase ou termo não soa natural quando passada do inglês para o português,
+### de forma a se encaixar no espaço disponível para exibição do texto. Isso inclui os termos técnicos usados por
+### profissionais da área de TI.
+### Parece haver um consenso de que alguns termos devem ser mantidos em inglês por soarem mais natural, mesmo no
+### meio de uma frase em português.
+### Por isso preferi manter alguns termos em inglês, pois pelo que pude perceber é o que os usuários ou
+### profissionais preferem. Não faz sentido traduzir termos que ninguém usa.
+### Claro, não sou dono da verdade, portanto se acharem que algo deve ser mudado (e que há um consentimento da
+### maioria), mude e inclua abaixo para manter a consistência.
+
+### Tudo isso vale para a tradução para o português brasileiro. Se você for de outro país que usa o português
+### e que acha que essas regras não lhe fazem sentido, por favor inclua sua tradução em uma linha separada.
+
+### Termos técnicos (incluindo palavras reservadas) mantidos em inglês, apesar de exisitirem softwares
+### que usam em português.
```

Comment:
Could you please remove it? We don't want to add so many comments to resource file. As far as I see it duplicates text written in the readme you wrote.

Reply:
And in other files too.

Reply:
As it was only a deletion task, I did the necessary changes on the web editor.
If there's anything else to be reviewed, I'd be glad to help.

Thanks again.

Reply:
Thank you. I'll move your readme to wiki and let you know.

Reply:
I put it here https://github.com/dbeaver/dbeaver/wiki/Localization#localization-proposals
If you want me to add your name as an author of such standartization proposals or to remove your nickname from link, please tell. Any other suggestions are welcomed.

Reply:
Hmm.. I see. We'll discuss what can be done with it